cmd/fyne/commands/release.go

```diff
@@ -56,20 +59,37 @@ func (r *releaser) Run(params []string) {
 func (r *releaser) afterPackage() error {
 	if util.IsAndroid(r.os) {
 		target := mobile.AppOutputName(r.os, r.packager.name)
-		return r.zipAlign(filepath.Join(r.dir, target))
+		apk := filepath.Join(r.dir, target)
+		err := r.zipAlign(apk)
+		if err != nil {
```

Comment:
nit:
```suggestion
		if err := r.zipAlign(apk); err != nil {
```

Reply:
Should this be added the contribution guide as "Declare variables in the smallest scope"?

Reply:
We need something as this has tripped up most of my recent committs.
However I don't think we can specify to always make it smallest or you will end up with very long, ugly lines.
Happy enough for you to add it in this context of error checking as that is quite common.

Reply:
Added, please take a look

Reply:
That is clear thanks. However my preference is that we don't add the "no" or "don't do this" versions if possible. If it stands alone with the "like this" version it feels overall a more positive doc.

Reply:
Fair enough, I'll make that change